src/components/EventTag/EventTag.stories.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withInfo } from "@storybook/addon-info";
+import { withKnobs, text, boolean } from "@storybook/addon-knobs";
+import EventTag from ".";
+import { peopleIcon } from "../../pages/EventPage/index";
+
+
+storiesOf("EventTag", module)
+  .addDecorator(
+    withInfo({
+      text: "This is a component renders an event Tag showing the event type or attendee number. The options for size are largeTag or smallTag"
+    })
+  )
+  .addDecorator(withKnobs())
+  .add("EventTag", () => {
+    return <EventTag 
+    icon={peopleIcon}
+    attendeeNumber={100}
+    size={"largeTag"}
```

Comment:
@kerry-okpere, not a P0 but you could write this as `attendeeNumber="largeTag"`. No need for the curly braces here